### PR TITLE
Update go.rb

### PR DESCRIPTION
**Pull request by syhily**
The repository url on google is inaccessible to chinese users. So please use origin Github url

### CODE REVIEW

File: Library/Formula/go.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@ class Go < Formula
   deprecated_option "cross-compile-common" => "with-cc-common"
 
   resource "gotools" do
-    url "https://go.googlesource.com/tools.git",
-    :revision => "69db398fe0e69396984e3967724820c1f631e971"
+    url "https://github.com/golang/tools",
```
Comment (bfontaine) [inline Library/Formula/go.rb]: Please add `.git` at the end of the URL.

Comment (syhily) [inline Library/Formula/go.rb]: @bfontaine OK, finished it. XD

**Comment by DomT4** [comment]
I seem to recall that we've actually rejected this change in the past because it differs from the official upstream links, but maybe we're fine with doing so now.

**Comment by mikemcquaid** [comment]
The revision shouldn't need to change. Can we add it as a `mirror` instead?

**Comment by DomT4** [comment]
We'll consider a mirror on a rebased formula, if that functionality works in this situation, Thanks.

**Pull request closed without merging**